generate_profiles/BuildXmlFiles.groovy

```diff
@@ -2,8 +2,8 @@ import groovy.xml.MarkupBuilder
 
 @Grapes([
     @Grab(group='com.google.code.findbugs'  , module='findbugs', version='3.0.1'),
-    @Grab(group='com.mebigfatguy.fb-contrib', module='fb-contrib', version='6.8.0'),
-    @Grab(group='com.h3xstream.findsecbugs' , module='findsecbugs-plugin', version='1.5.0')]
+    @Grab(group='com.mebigfatguy.fb-contrib', module='fb-contrib', version='7.0.0'),
```

Comment:
I was not able to specify version parameter as a constant as Groovy wants an inline constant.
Final static field was not good for it